deepinv/datasets/urban100.py

```diff
@@ -0,0 +1,135 @@
+from typing import Any, Callable
+import os
+
+from PIL import Image
+import torch
+
+from .utils import calculate_md5_for_folder, download_zipfile, extract_zipfile
+
+
+class Urban100HR(torch.utils.data.Dataset):
```

Comment:
I would remove the HR from the name

Reply:
I have a preference on HR since we are not exactly using the whole dataset (with all 11 variations).
Putting HR explicitly will make them know that they are not using the whole dataset.

Reply:
ok